test_install/CMakeLists.txt

```diff
@@ -7,6 +7,12 @@ find_package(Ginkgo REQUIRED
     # Alternatively, use `cmake -DCMAKE_PREFIX_PATH=<ginkgo_install_dir>` to specify the install directory
     )
 
+
+# Manage RPATH. This needs to be before the targets.
+set(CMAKE_SKIP_BUILD_RPATH OFF)
+set(CMAKE_SKIP_INSTALL_RPATH OFF)
+set(CMAKE_INSTALL_RPATH_USE_LINK_PATH ON)
```

Comment:
Do these need to be user configurable ?

Reply:
Not really, they need to always be set so that things are correct. The first two should always be off by default, that's only to make sure.

The last one says that we want the INSTALL_RPATH to be the one used for building: it makes sense here because we build against an already installed Ginkgo, so after installation the RPATH settings do not have to change, they *need* to be the same as what was used in the build step.